webservice/webservice.py

```diff
@@ -322,7 +339,85 @@ def get_manifest():
     # Get the response back
     logger.info("Creating the API response")
     response = es_td.transform_manifest(filters=filters)
-    # Return the excel file
+    if format == 'tsv':  # returns metadata as TSV file
+        return response
+    elif format == 'bdbag':  # returns presigned URL to a BDBag in S3
+        response_bag = bdbag_response(response)
+        if 'status' in response_bag:
+            http_status = response_bag['status']
+        else:
+            http_status = 200
+        return jsonify(response_bag), http_status
```

Comment:
You do not handle the case where an invalid format is specified. You need another else, and you should return a status of 400 (bad request). Probably `return '', 400`.

Reply:
I was looking trying to write the UI code, and for some reason it is already setting the format to `tarball`. I don't know where that comes from. In any case, to handle that and the case my comment describes, you need to do one of two things:

1. Treat everything that is not `bdbag` as tsv, e.g., make line 342 be `if format != 'bdbag'`
2. Add the test for `tarball`, e.g., change line 342 to be `if format == 'tsv' or format == 'tarball'` (then you still need a final else).

Whichever you pick, you should update the doc at the top accordingly.